tests/experimental/pydantic/schema/test_mutation.py

```diff
@@ -195,5 +189,5 @@ def create_user(self, input: CreateUserInput) -> Union[UserType, UserError]:
     assert result.errors is None
     assert result.data["createUser"].get("name") is None
     assert result.data["createUser"]["nameErrors"] == [
-        ("ensure this value has at least 2 characters")
+        "value_error.any_str.min_length: ensure this value has at least 2 characters"
```

Comment:
do we want to include the error code by default? this might make it harder to use the value of the error in the frontend 🤔